core/src/test/java/com/radixdlt/p2p/test/P2PTestNetworkRunner.java

```diff
@@ -188,6 +194,44 @@ private static Injector createInjector(
       throws ParseException {
     final var properties = RuntimeProperties.fromCommandLineArgs(new String[] {});
     return Guice.createInjector(
+        new AbstractModule() {
+          @Override
+          protected void configure() {
+            bind(AddressBookPersistence.class).to(BerkeleyAddressBookStore.class);
```

Comment:
Why are we using Berkeley here instead of Rocks / the default?

Did Rocks break? This feels like a scary change to me :).

Either this test should be changed to use FunctionalNodeModule if it's an integration test; or if that's too big a change, at least at a comment here to explain we're hacking in the deprecated address Berkeley stores for now because it's too disruptive to add a node rust environment here.

Reply:
I've tried to replace the implementation here, but got a very strange error:
```
com.google.inject.CreationException: Unable to create injector, see the following errors:

1) An exception was caught and reported. Message: called `Result::unwrap()` on an `Err` value: Error { message: "IO error: lock hold by current process, acquire time 1724758428 acquiring thread 6169243648: /var/folders/kq/xrwzyf4j3yq7pqnx_zkx9tym0000gn/T/junit5819788118170431666/state_manager/LOCK: No locks available" }
  at [unknown source]

2) [Guice/MissingConstructor]: No injectable constructor for type NodeRustEnvironment.

class NodeRustEnvironment does not have a @Inject annotated constructor or a no-arg constructor.

Requested by:
1  : NodeRustEnvironment.class(NodeRustEnvironment.java:75)
     at P2PTestNetworkRunner$1.configure(P2PTestNetworkRunner.java:198)
```
I'm going to add a comment and leave this part as is.